oxidation/libparsec_python/src/manifest.rs

```diff
@@ -395,6 +395,8 @@ impl FileManifest {
             &author_verify_key.0,
             &expected_author.0,
             expected_timestamp.0,
+            None,
```

Comment:
Shouldn't we remove this file now ?

Reply:
no, `manifest` is used by `local_manifest` which is used by `storage` & `file_operation`